apps/cookbook/src/app/page/side-nav/side-nav.component.html

```diff
@@ -103,6 +103,7 @@
       <li><a routerLink="showcase/flag" [routerLinkActive]="'is-selected'" class="side-nav__item">Flag</a></li>
       <li><a routerLink="showcase/icon" [routerLinkActive]="'is-selected'" class="side-nav__item">Icon</a></li>
       <li><a routerLink="showcase/progress-circle" [routerLinkActive]="'is-selected'" class="side-nav__item">Progress Circle</a></li>
+      <li><a routerLink="showcase/link" [routerLinkActive]="'is-selected'" class="side-nav__item">Link</a></li>
```

Comment:
It is called `Link` in Zeplin, and I think you would first look under 'L' and not 'T' if you were looking for something like this in the cookbook. 

So I think we should just rename the component to `Link` instead of `TextLink`.